runtime/include/qio/qio.h

```diff
@@ -546,7 +548,8 @@ qioerr qio_file_init_usr(qio_file_t** file_out, void* file_info,
 
 qioerr qio_file_open_access_usr(qio_file_t** file_out, const char* pathname, 
                                const char* access, qio_hint_t iohints, 
-                               const qio_style_t* style, void* fs_info,
+                               const qio_style_t* style,
+                               void* fs_info,
```

Comment:
Yeah, I know. I had (temporarily) added another argument. I think I like it to be multi-line, though it's a trivial change.